main.cpp

```diff
@@ -447,36 +449,36 @@ struct Point
     Point(const IntType& itx_, const IntType& ity_) : Point(static_cast<float>(static_cast<int>(itx_)), static_cast<float>(static_cast<int>(ity_))) {}
     Point(const DoubleType& dtx_, const DoubleType& dty_) : Point(static_cast<float>(static_cast<double>(dtx_)), static_cast<float>(static_cast<double>(dty_))) {}
     
-    Point& multiply(float m);
-    Point& multiply(FloatType& ftm);
-    Point& multiply(DoubleType& dtm);
-    Point& multiply(IntType& itm);
+    Point& operator*=(float m);
+    Point& operator*=(FloatType& ftm);
+    Point& operator*=(DoubleType& dtm);
+    Point& operator*=(IntType& itm);
```

Comment:
pass by const reference!
You have declared this function in such a way that it is possible to modify the object being passed in.
Unless you explicitly need to modify that object, you should always pass by const reference unless the object is a primitive or trivial type, such as `juce::String`, `juce::Rectangle<>`
- [ ] revise